### ISSUE TITLE
Should @spec syntax highlighting be more muted/subtle?

### ISSUE THREAD

**Issue by ryanwinchester** [fr1zle/vscode-elixir#118]
I personally find the `@spec` syntax highlighting to add too much visual noise to my code.

I find it harder to scan for specific functions I'm looking for, and it fights for attention with function definitions.

Does anybody else feel this way at all?

How hard would it be to make something similar-looking to _(shudders)_ PHP docblock which doesn't fight as much for attention but still easy to pick out relevant info?

![](http://drops.ryanwinchester.ca/0t1B0A2j3i3U/elixir-v-php-spec.png)

**Comment by h4cc** [comment]
I can recommend the highlighting the Elixir Plugin for IntelliJ IDEA @KronicDeth uses:

![intelij-elixir-highlighting](https://user-images.githubusercontent.com/2981491/35035165-c580bcfc-fb70-11e7-8109-b89da2cb6cfb.png)

**Comment by ryanwinchester** [comment]
I think, what I'm trying to say is that I think @spec should look closer in appearance to documentation than code.

Does anybody else think so?

**Comment by benperiton** [comment]
I tend to agree, but I'm not sure it should be the same coloring as documentation - as it is a different thing.

But not having the same highlighting as 'actual' code would be nice.

**Comment by dogweather** [comment]
Yes, this is an excellent idea. But wouldn't this be up to individual themes?

**Comment by KronicDeth** [comment]
@fr1zle I feel for you: people fighting about the default color schemes, when anyone can create and publish their own themes, has generated a lot of issues for me.

For any users that want to generate their own themes, I explain why I decided to make certain colors close or contrasting in https://github.com/KronicDeth/intellij-elixir/blob/master/COLOR_SCHEME_DESIGN.md.
